.github/workflows/build_n_deploy_prod_dagpenger.yaml

```
@@ -0,0 +1,54 @@
+name: Build and deploy dp-prosessering to prod
+
+on:
+  workflow_dispatch:
+
+env:
+  IMAGE: ghcr.io/${{ github.repository }}/familie-prosessering:${{ github.sha }}
```

Tror dette skal være tilstrekkelig
```suggestion
  IMAGE: ghcr.io/${{ github.repository }}:${{ github.sha }}
```